github/apps_installation.go

```diff
@@ -47,3 +50,38 @@ func (s *AppsService) ListRepos(ctx context.Context, opt *ListOptions) ([]*Repos
 
 	return r.Repositories, resp, nil
 }
+
+// AddRepo adds a single repository to an installation.
+//
+// GitHub API docs: https://developer.github.com/v3/apps/installations/#add-repository-to-installation
+func (s *AppsService) AddRepo(ctx context.Context, instID, repoID int) (*Repository, *Response, error) {
```

Comment:
I think we should call this method `AddRepository` rather than `AddRepo`. We already use the full "Repository" word in many other method names, e.g.:

- [`ActivityService.GetRepositorySubscription`](https://godoc.org/github.com/google/go-github/github#ActivityService.GetRepositorySubscription)
- [`ActivityService.DeleteRepositorySubscription`](https://godoc.org/github.com/google/go-github/github#ActivityService.DeleteRepositorySubscription)
- [`IssuesService.ListRepositoryEvents`](https://godoc.org/github.com/google/go-github/github#IssuesService.ListRepositoryEvents)